client/crypto-sdk-android/crypto-sdk/src/main/java/com/orbs/client/OrbsContract.java

```diff
@@ -0,0 +1,65 @@
+package com.orbs.client;
+
+
+import com.google.gson.Gson;
+import com.orbs.cryptosdk.Address;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+public class OrbsContract {
+  private final OrbsClient orbsClient;
```

Comment:
maybe now that we've separated between protocol (creating the JSON messages) and transport (the network), the relationship between Contract and Client should be inverted - the Client has a Contract instance which can only work as a codec, marshalling and unmarshalling the HTTP messages, with the Client object sending them over the wire? this also makes sense in terms of the contract test